README.md

```diff
@@ -36,15 +36,6 @@ The most important sources of information and activity are:
 **If you'd like clarification about anything**, don't hesitate to ask in the
 chat room or on the mailing list.
 
-Setting Up the Repo
```

Comment:
The bit about git configuration and `--prune` seems like it should be in the README if anywhere since it's much more likely to be read there.